app/full_text_search.py

```diff
@@ -37,6 +37,22 @@ def make_or_regexp(query_txt):
     regexp = '|'.join([re.escape(word) for word in query_words if word])
     return re.compile(regexp, re.I)
 
+def create_sort_expression(**kwargs):
+    """
+    Creates SortExpression's for ranking.
+    Args:
+        **kwargs: key=field_name, value=field_rank_value
+    Returns:
+        array of SortExpression
+    """
+    expressions = []
+    for field in kwargs:
+        expressions.append(appengine_search.SortExpression(
```

Comment:
It looks you need to use "count(field_name)" instead:

https://cloud.google.com/appengine/docs/python/search/sortexpressionclass
> The expression can simply be a field name, or some compound expression such as _score + count(likes) * 0.1 which will add the score from a scorer to a count of the values of a likes field weighted by a factor of 0.1.